Sources/Web3Auth/Types.swift

```diff
@@ -200,6 +200,7 @@ public struct W3AInitParams: Codable {
         self.mfaSettings = mfaSettings
         self.sessionTime = min(30 * 86400, sessionTime)
         self.originData = originData
+        self.dashboardUrl = URL(string: getDashBoardUrl(buildEnv: self.buildEnv))
```

Comment:
We don't account for dashboardURL taken in constructor, we always use the getDashBoardUrl. We should check if it's nil, or not and then use getDashBoardUrl. 

If it's not supposed to be exposed, we should remove from constructor params.